docs/topics/circuit-breaker-mission-basic-interaction-spring-boot-tomcat.adoc

```diff
@@ -0,0 +1,92 @@
+= Interact with the Unmodified {SpringBoot} Booster
+
+Once you have the {SpringBoot} booster deployed, you have the following services running:
+
+//{app-name} = springboot-cb
+
+`{app-name}-greeting`::
+Exposes the `api/greeting` HTTP endpoint that the service client calls to get a personalized greeting response.
+
+This endpoint calls the remote `api/name` endpoint to obtain the current state of the `{app-name}-name` service.
+
+If the remote endpoint is available, you receive the following greeting:
+----
+{"content":"Hello, World!"}
+----
+
+If service fails, you receive the predefined fallback response.
+----
+{"content":"Hello, Fallback!"}
+----
+
+`{app-name}-name`::
+Exposes the `api/name` HTTP endpoint that this application uses to simulate the failure of a service, and which returns
+
+* an HTTP Code `200` response when the service is available.
+* an HTTP Code `500` response when the service is unavailable.
+
+The following steps demonstrate how to verify the service availability, simulate a failure and receive a fallback response.
+
+NOTE: The following steps use the command line to interact with the service. Alternatively, you can use the web interface to perform the same steps.
+
+. Use `curl` to execute a `GET` request against the `{app-name}` service. You can also use a browser to do this.
++
+[source,bash,options="nowrap",subs="attributes"]
+----
+$ curl http://{app-name}-{project-name}.1ab5.starter-us-east-1.openshiftapps.com/api/greeting
+{"content":"Hello, World!"}
+----
+
+. Invoke the /api/killme endpoint and verify the availability of the /api/greeting endpoint shortly after that.
++
+Invoking the /api/killme endpoint simulates an internal service failure and triggers the OpenShift self-healing capabilities. When invoking /api/greeting after simulating the failure, the service should return a HTTP status 503.
```

Comment:
It _doesn't_ trigger OpenShift self-healing capabilities. It seems this is copied from the Health Check docs, but that's obviously something completely different :-)